src/main/resources/openapi/sqs/print-api-sqs-messaging.yaml

```diff
@@ -165,6 +176,21 @@ components:
         - sourceType
         - gssCode
 
+    RemoveCertificateMessage:
+      description: The SQS message sent to remove an individual Certificate record from the database and its corresponding photo from S3.
```

Comment:
Minor: If you add `title: RemoveCertificateMessage` then swagger ui will show it's name